lib/src/screens/exchange_trade/exchange_trade_page.dart

```diff
@@ -159,10 +155,13 @@ class ExchangeTradeState extends State<ExchangeTradeForm> {
                                   decoration: BoxDecoration(
                                       border: Border.all(
                                           width: 3,
-                                          color: Theme.of(context).extension<ExchangePageTheme>()!.qrCodeColor
-                                      )
+                                          color: Theme.of(context)
+                                              .extension<ExchangePageTheme>()!
+                                              .qrCodeColor)),
+                                  child: QrImage(
+                                    data: trade.inputAddress ?? fetchingLabel,
+                                    version: 14,
```

Comment:
we can consider making the default value null instead of 9 so that it will default to `QrVersions.auto`, what do you think?

Reply:
didn't realize there was an auto option, I'll make the change and be sure to test it